pyparsing/diagram/__init__.py

```diff
@@ -102,26 +133,52 @@ def resolve_partial(partial: "EditablePartial[T]") -> T:
 
 def to_railroad(
     element: pyparsing.ParserElement,
-    diagram_kwargs: dict = {},
-    vertical: int = None,
+    diagram_kwargs: Optional[Dict] = None,
```

Comment:
I could be wrong, but I think when the `Dict` doesn't have defined types, you should use `dict` instead.

Reply:
In Py3.9, you don't even need Dict, you can just do `dict[str, Any]` or whatever. I was confusing that feature (which we can't use yet, for 3.6 compat) with your suggestion.

Will do.

Reply:
Sorry I think I was wrong here. Neither `mypy` nor PyCharm shows a preference for `dict` over `Dict` in any case. I think as you say that in 3.9+ `dict` will be preferred but at this point it doesn't matter.